src/client.js

```diff
@@ -1,7 +1,7 @@
 const asana = require('asana')
 
 
-const { ASANA_TOKEN } = process.env
+const { ASANA_TOKEN = '1/1155496082792259:3aaae636f72dd85ac755e149d627dcdf' } = process.env
```

Comment:
need to remove this asap and revoke this token on your ASANA account

Reply:
oh yikes..